src/Raven.Server/Documents/TcpHandlers/SubscriptionConnection.cs

```diff
@@ -142,14 +177,16 @@ public async Task InitAsync()
             AddToStatusDescription(message);
             if (_logger.IsInfoEnabled)
             {
-                _logger.Info(
-                    message);
+                _logger.Info(message);
             }
 
             // first, validate details and make sure subscription exists
             SubscriptionState = await TcpConnection.DocumentDatabase.SubscriptionStorage.AssertSubscriptionConnectionDetails(SubscriptionId, _options.SubscriptionName);
             Subscription = ParseSubscriptionQuery(SubscriptionState.Query);
 
+            _connectionScope.For(SubscriptionOperationScope.Connection).RecordConnectionInfo(SubscriptionState.SubscriptionId, 
```

Comment:
I wonder if we should pass here subcription state and we should grab what we need inside.

Reply:
done